app/components/tables/headers/sort.js

```diff
@@ -15,6 +15,12 @@ export default class Sort extends Component {
         return 'caret down';
       }
     }
+    else {
+      //default sorting for starts-at column
+      if(kebabCase(this.column.valuePath)=='starts-at'){
```

Comment:
Codacy found an issue: [Operator '==' must be spaced.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6279483)